src/imginputfileconn.h

```diff
@@ -613,6 +650,9 @@ namespace dd
     int _scale_max = 1000;
     bool _keep_orig = false;
     std::string _interp = "cubic";
+#ifdef USE_CUDA_CV
+    bool _cuda = false;
```

Comment:
should not be this attribute value copied during ImgInputFileConn copy construction, somewhere around line 386?

Reply:
Ohlala, good catch, thanks!